### PR TITLE
fix: use name and link instead of linkType and url for external links

### LINKED ISSUE #1522: Adding URLs to the Edit Colony Details action type
## Description

Adding URLs via the Social links section in the 'Edit Colony Details' action type is broken. 

When adding the social links to the action, the all content added in the action do not get saved even when the transaction goes through.

If I do not add social links, the action content above saves once the transaction is made.  

image

### PR DESCRIPTION
**Pull request by bassgeta**
## Description

This PR fixes editing of social links.
The issue was that we used `linkType` and `url` as the values, which wasn't the case anymore.
Also fixed a minor CSS alignment on the links modal.

I updated our version of apollo, so we could strip `__typename` on the mutation and so that the `haveExternalLinksChanged` flag isn't always true when editing details. 

**Changes** 🏗
* renamed the attributes accordingly
* updated apollo client version

Resolves  #1522

### CODE REVIEW

File: src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/hooks.ts [hunk 1]
```diff
@@ -61,12 +61,10 @@ export const useEditColonyDetails = (
...
-            externalLinks: payload.externalLinks.map(
-              ({ linkType, ...link }) => ({
-                ...link,
-                name: linkType,
-              }),
-            ),
+            externalLinks: payload.externalLinks.map(({ name, link }) => ({
+              link,
+              name,
+            })),
```
Comment (jakubcolony) [inline src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/hooks.ts:67]: Do we actually need that map?

**Review by jakubcolony — APPROVED** [review]
Looks good, nice work 👍

File: src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/hooks.ts (hunk 1 above)
Comment (bassgeta) [reply src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/hooks.ts:67]: huh good point, I don't really think we do :D

**Review by jakubcolony — DISMISSED** [review]

File: src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/hooks.ts (hunk 1 above)
Comment (chmanie) [reply src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/hooks.ts:63]: You only might get extra properties but that shouldn't matter

File: src/components/v5/common/Fields/RadioButtons/TileRadioButtons/TileRadioButtons.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ function TileRadioButtons<TValue = string>({
               border
               transition-all
               rounded-lg
-              align-center
+              items-center
```
Comment (chmanie) [inline src/components/v5/common/Fields/RadioButtons/TileRadioButtons/TileRadioButtons.tsx:30]: Nice find

**Review by chmanie — COMMENTED** [review]
Remove that map and get an approval for free :)

**Review by jakubcolony — DISMISSED** [review]

**Review by rdig — APPROVED** [review]
Yup! :+1:

**Review by jakubcolony — APPROVED** [review]

**Review by ArmandoGraterol — APPROVED** [review]
LGTM 👌

**Pull request merged**